common/source_model/src/test/scala/weco/catalogue/source_model/sierra/rules/SierraItemAccessTest.scala

```diff
@@ -846,7 +846,123 @@ class SierraItemAccessTest
+      val displayreservation =
+        "Locked filing cabinet, disused lavatory with a sign saying 'Beware of The Leopard'"
+      val itemData = createSierraItemDataWith(
+        fixedFields = Map(
+          "79" -> FixedField(
+            label = "LOCATION",
+            value = "exres",
+            display = "On Exhibition")
+        ),
+        varFields = List(
+          VarField(fieldTag = "r", displayreservation)
+        )
+      )
+
+      val (ac, _) = SierraItemAccess(
+        location = Some(LocationType.OnExhibition),
+        itemData = itemData
+      )
+
+      ac shouldBe AccessCondition(
+        method = AccessMethod.NotRequestable,
+        note = Some(displayreservation)
+      )
+    }
+    it("can show multiple Reserves Notes") {
+      val itemData = createSierraItemDataWith(
+        fixedFields = Map(
+          "79" -> FixedField(
+            label = "LOCATION",
+            value = "exres",
+            display = "On Exhibition")
+        ),
+        varFields = List(
+          VarField(fieldTag = "r", "in the bottom of a locked filing cabinet"),
+          VarField(fieldTag = "r", "stuck in a disused lavatory"),
+          VarField(
+            fieldTag = "r",
+            "with a sign on the door saying 'Beware of The Leopard'")
+        ),
+      )
+
+      val (ac, _) = SierraItemAccess(
+        location = Some(LocationType.OnExhibition),
+        itemData = itemData
+      )
 
+      ac shouldBe AccessCondition(
+        method = AccessMethod.NotRequestable,
+        note = Some(
+          "in the bottom of a locked filing cabinet<br />" +
+            "stuck in a disused lavatory<br />" +
+            "with a sign on the door saying 'Beware of The Leopard'"
+        )
+      )
+    }
+    it("Only shows substantive content from Reserves Notes") {
+      val itemData = createSierraItemDataWith(
+        fixedFields = Map(
+          "79" -> FixedField(
+            label = "LOCATION",
+            value = "exres",
+            display = "On Exhibition")
+        ),
+        varFields = List(
+          VarField(fieldTag = "r", "25-12-22 ON RESERVE FOR Beware of the Leopard"),
+          VarField(fieldTag = "r", "25-12-22 OFF RESERVE FOR Beware of the Leopard CIRCED 2 TIMES"),
+          VarField(fieldTag = "r", "In a locked filing cabinet")
+        )
+      )
+
+      val (ac, _) = SierraItemAccess(
+        location = Some(LocationType.OnExhibition),
+        itemData = itemData
+      )
+      // To avoid confusion for staff, and in the interest of completeness, notes that are identical
+      // after removing the non-interesting content are preserved.
+      // This scenario *should* not be encountered.
+      //  - Something that is On Exhibition is not expected to have an off reserve entry.
+      //  - Something that has both an on and an off reserve entry is not expected to be On Exhibition.
+      // However, it is something that *could* happen.
```

Comment:
Maybe worth a sentence about what we should do if it does?

"If this happens, we should get the record corrected in Sierra, rather than fixing it in our transformation logic"